.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  pull_request:
+  push: 
+    branches: 
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '6'
+      - run: yarn
+      - run: bin/check_required_files_present
+      - run: bin/check_optional
```

Comment:
```suggestion
```

This is about to be removed in the `.travis.yml` via #1677 so probably shouldn't be added here.